src/System.Device.Gpio/System/Device/Gpio/Drivers/LibgpiodDriver.cs

```diff
@@ -0,0 +1,190 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+using System.Threading;
+
+namespace System.Device.Gpio.Drivers
+{
+    public class LibgpiodDriver : GpioDriver
+    {
+        private const string GpioGet = "gpioget";
+        private const string GpioChip0 = "gpiochip0 ";
```

Comment:
This driver is supposed to be generic, so even though in the raspberry pi we only allow the chip0, we should check what happens on different borads.

Reply:
yeah agree, I will think about how to check other boards, I was thinking to add chip select option but you know GpioDriver interface doesn't such method. for me its better to have option for chip select instead of trying to check all board versions chip id

Reply:
I agree.  We should probe and enumerate the gpiochipX entries on the machine at runtime

Reply:
This would be added in pinvoked libgpiod implementation